port_drayage_plugin/src/port_drayage_worker.cpp

```diff
@@ -121,8 +121,8 @@ namespace port_drayage_plugin
 
         // Add current vehicle location (latitude and longitude)
         ptree location;
-        location.put("latitude", _current_gps_position.latitude * 10000000); // Convert degrees to 1/10 microdegrees
-        location.put("longitude", _current_gps_position.longitude * 10000000); // Convert degrees to 1/10 microdegrees
+        location.put("latitude", _current_gps_position.latitude); 
```

Comment:
I can't add a comment on the line, but where this is initialized I think it might be wise (if extremely cautious) to either check, and make sure that this `_current_gps_position` struct has meaningful data (we've gotten a pose update after having a georeference) OR at least initializing it explicitly to `{0.0, 0.0}` where it's defined as a member variable or as a default value for the struct type.

Reply:
Good catch, I've initialized the latitude and longitude values within the LatLonCoordinate struct to 0.0 to address this.